mantidproject/mantid
# Indirect Data Analysis - Remove IndirectDataAnalysis.py

`IndirectDataAnalysis.py` has now been removed from the code base. The remaining function has been moved to the `IqtFitMultiple` algorithm as it is the only place that still used the function.

**To test:**
- Ensure the test for `IqtFitMultiple` still passes
- Ensure `IndirectDataAnalysis.py` has been removed from the `scripts/Inelastic` directory 

Fixes #16532

_Does not need to be in the release notes. As users do not use this script directly_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
